### ISSUE TITLE
"This was me" not working when IG detects a suspicious login attempt

### ISSUE THREAD

**Issue by astropingo** [instaloader/instaloader#1963]
When trying to login with Instaloader, IG detects it as an unusual login attempt. Instaloader gives me a checkpoint required point. There I have two buttons, "This wasn't me" and "This was me". But if I click on "this was me", instead of logging me in it loads the page to reset the password. No matter what I try, I can't get IG to trust the login attempt.

Is anyone else having this problem?

**Comment by JunedKhan101** [comment]
I was facing this issue a while back, it was forcing me change my password whenever I used instaloader.
I just tried using it again but it doesn't seem to be occuring now.
I believe if i used it over and over again it will eventually think that I'm a suspicious user trying to break into someone's account.
Right now I haven't figured it out how to circumvent this issue except to use a VPN of some sort to bypass suspicious activity detection temporarily.

See https://github.com/instaloader/instaloader/issues/1635#issuecomment-1304812276 and #1954
